merkle/index.go

```diff
@@ -0,0 +1,78 @@
+package merkle
+
+import "sort"
+
+// siblingIndecies returns indecirs of sibling elements
+func siblingIndecies(idxs []uint64) []uint64 {
```

Comment:
maybe call it `mapSiblingIndex`?

Reply:
This does not going to map anything, it returns slice of all sibling indices

Reply:
Ok, thanks!